src/docs/implementations/new-relic.adoc

```diff
@@ -1,113 +1,143 @@
 = Micrometer New Relic
-Jon Schneider <jschneider@pivotal.io>
+New Relic Open Source <opensource@newrelic.com.>
 :toc:
 :sectnums:
 :system: new-relic
 
-New Relic offers a dimensional monitoring system product called Insights with a full UI and a query language called NRQL. New Relic Insights operates on a push model. Some features of NRQL assume that Insights receives a distinct event payload for every timing, count, etc. Micrometer instead ships aggregates at a prescribed interval, allowing your app's throughput to scale without concern for event propagation to Insights becoming a bottleneck.
+New Relic is a powerful observability platform with Micrometer
```

Comment:
superlatives suck. All of the platforms micrometer supports are (differently) powerful. ;)